templates/webapi/main/index.html.ep

```diff
@@ -83,20 +83,11 @@
 
             <div class="form-group">
                 <strong>
-                    Auto refresh
-                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically</p>') %>
+                    Auto refresh in (seconds)
+                    <%= help_popover('Help for <i>Auto refresh</i>' => '<p>Allows to update the data of this page automatically. To stop the automatic refresh select 0</p>') %>
                 </strong>
+                <input type="number" class="form-control" name="interval" value="0" id="filter-limit-builds">
```

Comment:
Also please use the same format as for the input fields above, i.e. `<input/>` first, then label, then `help_popover` but with imperative, e.g. s/Allows to update the data of this page/Update this page/